File: CoreJava/College-app/src/com/techlabs/college/Student.java
package com.techlabs.college;

import java.util.*;

public class Student extends Person  {
	private int id;
	private String address;
	private Date dob;
	private Branch branch;

	public Student(int id, String address, Date dob, Branch branch) {
		super();
		this.id = id;
		this.address = address;
		this.dob = dob;
		this.branch = branch;
	}

	public int getId() {
		return id;
	}

	public String getAddress() {
		return address;
	}

	public Date getDob() {
		return dob;
	}

	public Branch getBranch() {
		return branch;
	}
}

File: DotNET/C#/CollegeApp/CollegeApp/Student.cs
using System;

namespace CollegeApp
{
    class Student
    {
        private int _id;
        private String _address;
        private DateTime _dob;
        private Branch _branch;
        
        public Student(int id,String address, DateTime dob,Branch branch)
        {
            _id = id;
            _address = address;
            _dob = dob;
            _branch = branch;
        }

        public int Id
        {
            get
            {
                return _id;
            }
        }

        public String Address
        {
            get
            {
                return _address;
            }
        }

        public DateTime Dob
        {
            get
            {
                return _dob;
            }
        }

        public Branch Branch
        {
            get
            {
                return _branch;
            }
        }




    }
}
